src/sequences/reducer.ts

```diff
@@ -37,11 +39,11 @@ export let sequenceReducer = generateReducer<SequenceReducerState>(initialState)
         let current_sequence = state
             .all[state.current] || populate(state);
         let { step } = action.payload;
-        step.position = step.position || current_sequence.steps.length;
+        step.position = step.position || current_sequence.body.length;
```

Comment:
No longer required. We can remove all refs to `step.position`. Might as well pull it out of the interface, too.